src/gaussiannumbers/GaussianNumberTypes.jl

```diff
@@ -3,7 +3,8 @@
 struct ZZiRing <: Ring
 end
 
-const FlintZZi = ZZiRing()
+const ZZi = ZZiRing()
+GaussianIntegers() = ZZi
```

Comment:
I'd be tempted to remove this, too, but it is mentioned in the docs *sigh*. But perhaps we can at least rename it to `gaussian_integers` to fit our naming conventions (I guess we need to keep the old name around for a bit, too)